src/content/en/2022/markup.md

```diff
@@ -1,22 +1,555 @@
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Doctype</th>
+        <th>desktop</th>
+        <th>mobile</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>`html`</td>
+        <td class="numeric">88.1%</td>
+        <td class="numeric">90.0%</td>
+      </tr>
+      <tr>
+        <td>`html -//w3c//dtd xhtml 1.0 transitional//en http://www.w3.org/tr/xhtml1/dtd/xhtml1-transitional.dtd`</td>
+        <td class="numeric">4.7%</td>
+        <td class="numeric">3.9%</td>
+      </tr>
+      <tr>
+        <td></td>
+        <td class="numeric">3.0%</td>
+        <td class="numeric">2.7%</td>
+      </tr>
+      <tr>
+        <td>`html -//w3c//dtd xhtml 1.0 strict//en http://www.w3.org/tr/xhtml1/dtd/xhtml1-strict.dtd`</td>
+        <td class="numeric">1.2%</td>
+        <td class="numeric">1.1%</td>
+      </tr>
+      <tr>
+        <td>`html -//w3c//dtd html 4.01 transitional//en http://www.w3.org/tr/html4/loose.dtd`</td>
+        <td class="numeric">0.9%</td>
+        <td class="numeric">0.6%</td>
+      </tr>
+      <tr>
+        <td>`html -//w3c//dtd html 4.01 transitional//en`</td>
+        <td class="numeric">0.4%</td>
+        <td class="numeric">0.4%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>
+    {{ figure_link(
+      caption="Doctype usage.",
+      sheets_gid="1535288056",
+      sql_file="doctype.sql",
+    ) }}
+  </figcaption>
+</figure>
+
+Let's start with doctypes—which one is the most popular? But you know the answer to this one: It's the short, simple, boring standard HTML doctype, that is, `<!DOCTYPE html>`.
+
+{{ figure_markup(
+  content="90%",
+  caption="Mobile using the standard HTML5 doctype.",
+  classes="big-number",
+  sheets_gid="1535288056",
+  sql_file="doctype.sql",
+) }}
+
+90% of all mobile pages use it—as the mobile data set is largest, this chapter will usually work with that data. Next most popular is XHTML 1.0 Transitional (3.9%, [down from 4.6% in 2021](../2021/markup#doctypes)). After that it's no doctype being set at all at 2.7%, up from 2.5% last year.
+
+### Compression
+
+{{ figure_markup(
+  image="content-encoding.png",
+  caption="HTML document content-encoding.",
+  description="Stacked bar chart, showing 28% of desktop and mobile HTML documents are being compressed with Brotli, 58% of desktop and mobile documents are being compressed with Gzip, and 14% of desktop and 13% of mobile HTML documents are not being compressed at all.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRjXjOczLu9q3mcQ-UFLUOmZgefGNCPVYeEwo4cDxQTymgmD_1D5dbZ728Mz1SkEMZHxQgwcWmLjkgx/pubchart?oid=1176900527&format=interactive",
+  sheets_gid="1434175283",
+  sql_file="content_encoding.sql",
+) }}
+
+Are HTML documents being compressed? How many? How? 86% of them are—with 58% (down 5.8% since last year) overall being gzip-compressed, and 28% (up 6.1%) being compressed using Brotli. To summarise slightly more documents are being compressed, and compressed more effectively.
```

Comment:
Yesh Web Almanac is American English.  But it's difficult for me to override 44 years of learning :-)

```suggestion
Are HTML documents being compressed? How many? How? 86% of them are—with 58% (down 5.8% since last year) overall being gzip-compressed, and 28% (up 6.1%) being compressed using Brotli. To summarize, slightly more documents are being compressed, and compressed more effectively.
```